packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -342,6 +342,7 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
       policyResponseTroubleshooting: {
         full_disk_access: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/deploy-elastic-endpoint.html#enable-fda-endpoint`,
         macos_system_ext: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/deploy-elastic-endpoint.html#system-extension-endpoint`,
+        linux_deadlock: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/deploy-elastic-endpoint.html`,
```

Comment:
this is just a placeholder, will update when new doc link is available.

Reply:
Troubleshooting docs for this will likely go here (the page is already published so we'll just be adding to it):
```suggestion
        linux_deadlock: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/ts-management.html#ts-endpoints`,
```

Reply:
Will there be a way to link to the specific troubleshooting section for linux deadlock avoidance? Overtime I imagine we'll add other troubleshooting tips.

Reply:
@ferullo Good idea, yes there will be a unique anchor for that particular section, which we can use in the URL. Until we publish the new section, the full URL will be broken, but most browsers will just go to the main page if they can't find an anchor reference. So something more like this:
```suggestion
        linux_deadlock: `${ELASTIC_WEBSITE_URL}guide/en/security/${DOC_LINK_VERSION}/ts-management.html#linux-deadlock`,
```

Reply:
@lcawl do we need to add the `linux_deadlock` keyword and link to the [Security solution keywords](https://github.com/elastic/kibana/blob/main/packages/kbn-doc-links/src/get_doc_links.ts#L338) in the `get_doc_links.ts` file?

Reply:
@nastasha-solomon I think that's what this suggested change is doing -- adding the keyword and link to the file:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/87339667/180803405-c05941f3-9a53-4d6e-bd76-1bcf3887e5d7.png">

Reply:
link has been updated